app/src/main/java/com/example/weatherapp/models/utils/DataFormatter.kt

```diff
@@ -39,7 +39,20 @@ fun Long.formatHour(resourceProvider: ResourceProvider, timeOffset: Int?): Strin
     timeOffset?.let {
         val zoneOffset = ZoneOffset.ofTotalSeconds(timeOffset)
         val dateTime = LocalDateTime.ofEpochSecond(this, 0, zoneOffset)
-        format = resourceProvider.getString(R.string.hour_format, dateTime.hour.toString())
```

Comment:
Check how to format date and time with `SimpleDateFormat`

Reply:
Resolved with DateTimeFormatter